src/modules/ipfs/ipfs.service.ts

```diff
@@ -64,10 +68,7 @@ export class IPFSService {
     }
 
     try {
-      await this.pinsQueue.add(
-        PIN_CLAIM_JOB_NAME,
-        JSON.stringify({ cid, claim })
```

Comment:
Do you recall why `stringify` was used initially? From what I can from the [NestJs Bull queue docs](https://docs.nestjs.com/techniques/queues) it seems fine to use a non-stringified object, but it's not clear to me why we didn't use the object directly in the first place 🤔

Reply:
No, I don't really remember reason